### ISSUE TITLE
Error when trying to setFieldLocales 

### ISSUE THREAD

**Issue by libbysharf** [facebook/facebook-java-ads-sdk#145]
Hi, I have the following code:
	public static class AdSetLocations implements AdSetAudienceConfiguration {
		private final List<Long> language;

		public AdSetLocations(List<Long> language) {
			this.language = language;
		}

		public void configure(Targeting targeting) {
			**targeting.setFieldLocales(language);**
		}
         }

	private Targeting createTargeting() {
		Targeting targeting = new Targeting();

		configurations.forEach(c -> c.configure(targeting));

		return targeting;
	}

	public void configure(AdAccount.APIRequestCreateAdSet adSet) {
		adSet.setTargeting(createTargeting());
	}

	private void configureAdSet(AdAccount.APIRequestCreateAdSet adSet, FacebookAdSetParams params) throws AutomatorApiException {
		adSet.setCampaignId(params.getCampaignId());

		adSet.setName(params.getName());

		adSet.setPacingType(STANDARD_PACING);

		params.getStatus().configure(adSet);

		// (Optional) Optimize for a Conversion
		params.getOptimization().configure(adSet);

		params.getBudgetAndSchedule().configure(adSet);

		params.getTargeting().configure(adSet);

		params.getDelivery().configure(adSet);
	}

	public Identifier create(FacebookAdSetParams params) throws AutomatorApiException {
		AdAccount adAccount = new AdAccount(params.getAccountId(), context);

		AdAccount.APIRequestCreateAdSet adSet = adAccount.createAdSet();

		// reserved ad set
		if (params instanceof ReachAndFrequencyAdSetParams) {
			Identifier predictionId = createAdSetPrediction((ReachAndFrequencyAdSetParams) params, adAccount);
			Long status = getPredictionStatus(predictionId);
			if (status != 1) {
				throw ExceptionFactory.from("Reach And Frequency parameters wrong (Facebook Status: " + status + "). " +
						"Please refer to the documentation - https://developers.facebook.com/docs/marketing-api/reachandfrequency/v2.8");
			}

			Identifier reservationId = reservePrediction(predictionId, adAccount);

			configureAdSet(adSet, params);

			adSet.setRfPredictionId(reservationId.getId());
		}
		// auction ad set
		else {
			configureAdSet(adSet, params);
		}

		try {
			AdSet response = adSet.execute();

			return Identifier.of(response.getId());
		} catch (APIException e) {
			LOGGER.debug("API call failed: {}", params.toString());
			throw ExceptionFactory.from(e);
		}
	}

To summarize this code, I'm trying to set fields in the targeting object and set it as the adset's targeting.
If I comment out the line in bold (targeting.setFieldLocales(language);) everything works fine and the adset is created with its targeting, but if I leave it there, I get this error:
{"error":{"message":"Invalid parameter","type":"OAuthException","code":100,"error_subcode":1487079,"is_transient":false,"error_user_title":"Invalid Targeting Spec","error_user_msg":"The specified targeting spec is not valid because: The field languages is not a valid target spec field","fbtrace_id":"Br0lL+tQgLc"}}

And the adset is not created.
Is this a bug? please advise, I need this urgently...
